resolwe_bio/descriptors/reads_descriptorschema.yaml

```diff
@@ -0,0 +1,20 @@
+- model: flow.descriptorschema
```

Comment:
Rename the file to reads.yaml. The file is in the descriptors folder so it is obvious it contains a descriptor schema.